lib/src/st2110/st_video_transmitter.c

```diff
@@ -457,6 +458,94 @@ static int video_trs_tsc_tasklet(struct mtl_main_impl* impl,
   return MT_TASKLET_HAS_PENDING;
 }
 
+static int video_trs_launch_time_tasklet(struct mtl_main_impl* impl,
+                                         struct st_tx_video_session_impl* s,
+                                         enum mtl_session_port s_port) {
+  unsigned int bulk = s->bulk;
+  struct rte_ring* ring = s->ring[s_port];
+  int tx = 0;
+  unsigned int n;
+  uint64_t i;
+  uint64_t target_ptp;
+  uint16_t port_id = s->port_id[s_port];
+  struct mt_interface* inf = mt_if(impl, port_id);
+  struct mt_ptp_impl* ptp = mt_get_ptp(impl, port_id);
+
+  if (!ptp->phc2sys.stat_sync) {
+    return MT_TASKLET_ALL_DONE;
+  }
+
+  /* check if any inflight pkts in transmitter */
+  if (s->trs_inflight_num[s_port] > 0) {
+    tx = mt_txq_burst(s->queue[s_port],
```

Comment:
Use the wrapper func `video_trs_burst` instead. We has other important logical in the wrapper.